### PR TITLE
Update cw_ownable README with instantiate and query setup process.

### PR DESCRIPTION
**Pull request by 0xekez**

### CODE REVIEW

**Comment by 0xekez** [comment]
requesting review from @larry0x 🙏

File: packages/ownable/README.md [hunk 1]
```diff
@@ -58,6 +77,55 @@ pub fn execute(
...
+use cosmwasm_std::{entry_point, Deps, Env, Binary};
+use cw_ownable::{cw_serde, get_ownership};
+
+#[entry_point]
+pub fn query(deps: Deps, env: Env, msg: QueryMsg) -> StdResult<Binary> {
+    match msg {
+        QueryMsg::Ownership {} => to_binary(&get_ownership(deps.storage)?),
+		_ => unimplemneted!(),
+    }
+	Ok(Binary::default())
```
Comment (larry0x) [inline packages/ownable/README.md]:
```rust
Ok(Binary::default())
```

Is this line needed?

**Comment by larry0x** [comment]
I made some simple changes. Merging now

File: packages/ownable/README.md (hunk 1 above)
Comment (0xekez) [reply packages/ownable/README.md]: lol great catch, thanks :)

**Pull request merged**